### PR TITLE
deprecatedapirequest crd

### PR DESCRIPTION
**Pull request by sanchezl**
- bump(*) - apiserver.openshift.io_deprecatedapirequests.yaml
- add deprecatedapirequests crd manifest
- add deprecatedapirequests to relatedObjects

### CODE REVIEW

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -132,6 +132,7 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 			"v4.1.0/kube-apiserver/localhost-recovery-sa.yaml",
 			"v4.1.0/kube-apiserver/localhost-recovery-token.yaml",
 			"v4.1.0/kube-apiserver/audit-policies-cm.yaml",
+			"v4.1.0/kube-apiserver/apiserver.openshift.io_deprecatedapirequests.yaml",
 		},
 		(&resourceapply.ClientHolder{}).WithKubernetes(kubeClient),
```
Comment (deads2k) [inline pkg/operator/starter.go:137]: don't you need to add the apiextensions client?

**Comment by deads2k** [comment]
This PR appears to cause a panic and does not create the CRD it attempts to.

File: pkg/operator/starter.go (hunk 1 above)
Comment (sanchezl) [reply pkg/operator/starter.go]: Done.

**Comment by deads2k** [comment]
crd is now created

/lgtm

**Comment by deads2k** [comment]
rebased in https://github.com/openshift/cluster-kube-apiserver-operator/pull/1102 for merge tonight

**Comment by sanchezl** [comment]
superseded by https://github.com/openshift/cluster-kube-apiserver-operator/pull/1102

**Pull request closed without merging**